CesiumGltf/include/CesiumGltf/SamplerUtil.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
```

Comment:
Can this be renamed to `SamplerUtility.h` to be stylistically consistent with the other files? Same for `SamplerUtil.cpp` -> `SamplerUtility.cpp`.